### PR TITLE
App responsiveness and minor features

### PR DESCRIPTION
**Pull request by therajatg**
1. styled the related video page.
2. made the 404 page
3. made the carousel (HeroContent.js) for the home page.
4. added title and favicon
5. 1. made the app responsive.
6. introduced toasts on login and playlist operations.
7. styled the HeroContent
8. styled navbar
9. modified VideoCard to show "Remove from history" option and deleted the HistoryCard.
10. In authContext used the localStorage so that even after refresh user remains logged in.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
-@import url("https://fonts.googleapis.com/css2?family=Arvo&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Pacifico&display=swap");
+/* @import url("https://fonts.googleapis.com/css2?family=Arvo&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Pacifico&display=swap"); */
```
Comment (prathmeshjagtap) [inline src/App.css]: you can remove commented code

File: src/frontend/components/heroContent/heroContent.module.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.carousel {
+  text-align: center;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  /* margin-top: -3rem; */
+  margin: 0;
+  padding: 0;
+  /* height: 20vh; */
```
Comment (prathmeshjagtap) [inline src/frontend/components/heroContent/heroContent.module.css]: remove commented code if not needed

**Review by prathmeshjagtap — APPROVED** [review]

File: src/App.css [hunk 3]
```diff
@@ -16,13 +17,23 @@
   --card-button-background-hover-color: grey;
   --nav-side-padding: 1rem;
   --body-side-padding: 1rem;
-  --general-font: "Arvo", serif;
-  --stylish-font: "Pacifico", cursive;
+  /* --general-font: "Arvo", serif;
```
Comment (parulgupta2001) [inline src/App.css]: remove commented code

File: src/frontend/components/playlistFolder/playlistFolder.module.css [hunk 4]
```diff
@@ -29,11 +46,13 @@
 .playlistAction {
   background-color: var(--nav-color);
   color: var(--text-color);
-  font-weight: 600;
+  /* font-weight: 600; */
```
Comment (parulgupta2001) [inline src/frontend/components/playlistFolder/playlistFolder.module.css:49]: remove commented code

**Review by parulgupta2001 — APPROVED** [review]

File: src/App.css (hunk 3 above)
Comment (therajatg) [reply src/App.css]: Done

File: src/frontend/components/heroContent/heroContent.module.css (hunk 2 above)
Comment (therajatg) [reply src/frontend/components/heroContent/heroContent.module.css]: Removed

**Pull request merged**